CMakeLists.txt

```diff
@@ -299,6 +299,14 @@ if(TESTS)
     # Hippomocks library
     add_subdirectory(${colobot_SOURCE_DIR}/lib/hippomocks)
 
+    include(FetchContent)
+    FetchContent_Declare(
+      cpptrace
+      GIT_REPOSITORY https://github.com/jeremy-rifkin/cpptrace.git
+      GIT_TAG        v0.7.3 # <HASH or TAG>
```

Comment:
TODO:

https://cmake.org/cmake/help/latest/module/FetchContent.html#command:fetchcontent_declare

> Where contents are being fetched from a remote location and you do not control that server, it is advisable to use a hash for GIT_TAG rather than a branch or tag name. A commit hash is more secure and helps to confirm that the downloaded contents are what you expected.

Reply:
I think I should use a submodule to be consistent with how other dependencies are handled